src/app/space/add-app-overlay/add-app-overlay.component.spec.ts

```diff
@@ -266,7 +266,7 @@ describe('AddAppOverlayComponent', () => {
 
     it('validate Project Name to be truthy', () => {
```

Comment:
Test name no longer matches the test assertion.
You should keep a valid name test and move this assertion to the falsy test case.

Reply:
I fixed it. Please review.